src/server/generic_family.cc

```diff
@@ -1397,12 +1406,13 @@ void GenericFamily::Register(CommandRegistry* registry) {
             << CI{"ECHO", CO::LOADING | CO::FAST, 2, 0, 0, 0}.HFUNC(Echo)
             << CI{"EXISTS", CO::READONLY | CO::FAST, -2, 1, -1, 1}.HFUNC(Exists)
             << CI{"TOUCH", CO::READONLY | CO::FAST, -2, 1, -1, 1}.HFUNC(Exists)
-            << CI{"EXPIRE", CO::WRITE | CO::FAST, 3, 1, 1, 1}.HFUNC(Expire)
-            << CI{"EXPIREAT", CO::WRITE | CO::FAST, 3, 1, 1, 1}.HFUNC(ExpireAt)
+            << CI{"EXPIRE", CO::WRITE | CO::FAST | CO::CUSTOM_JOURNAL, 3, 1, 1, 1}.HFUNC(Expire)
+            << CI{"EXPIREAT", CO::WRITE | CO::FAST | CO::CUSTOM_JOURNAL, 3, 1, 1, 1}.HFUNC(ExpireAt)
```

Comment:
I should better call this `NO_AUTOJOURNAL` or something else - it should indicate that the tx shouldn't log the operation automatically